c/sudt_contracts.h

```diff
@@ -161,11 +208,20 @@ int transfer_to_any_sudt(gw_context_t* ctx,
     return ERROR_TRANSFER_TO_ANY_SUDT;
   }
 
-  evmc_address from_address = *((evmc_address *)(input_src + 32 + 12));
-  evmc_address to_address = *((evmc_address *)(input_src + 64 + 12));
-  ret = sudt_transfer(ctx, sudt_id, POLYJUICE_SHORT_ADDR_LEN, from_address.bytes, to_address.bytes, amount);
+  gw_reg_addr_t from_addr = {0};
+  from_addr.reg_id = GW_DEFAULT_ETH_REGISTRY_ACCOUNT_ID;
+  memcpy(from_addr.addr, input_src + 32 + 12, ETH_ADDRESS_LEN);
+  from_addr.addr_len = ETH_ADDRESS_LEN;
+
+  gw_reg_addr_t to_addr = {0};
+  to_addr.reg_id = GW_DEFAULT_ETH_REGISTRY_ACCOUNT_ID;
+  memcpy(to_addr.addr, input_src + 64 + 12, ETH_ADDRESS_LEN);
+  to_addr.addr_len = ETH_ADDRESS_LEN;
```

Comment:
use function to build `gw_reg_addr_t` value.

Reply:
Please search all usage of `GW_DEFAULT_ETH_REGISTRY_ACCOUNT_ID`, and replace the code to function call.

Reply:
https://github.com/nervosnetwork/godwoken-polyjuice/pull/138/commits/0ea8e4e4b8ded3dfb992293af381f3699f495900